webapp/src/main/java/org/opentestsystem/rdw/admin/model/GroupQuery.java

```diff
@@ -0,0 +1,65 @@
+package org.opentestsystem.rdw.admin.model;
+
+import java.util.Set;
+
+public class GroupQuery {
+    private int schoolId;
```

Comment:
Should schoolId and schoolYear be non-primitive types so they can be null, i.e. unspecified in the query?